js/autotrack/touchables.js

```diff
@@ -1,16 +1,20 @@
 import { getBaseComponentProps } from './common';
 
 export const autotrackPress = track => (eventType, componentThis, event) => {
+  const autotrackProps = getBaseComponentProps(componentThis);
```

Comment:
Seems like it's a common occurrence.  Maybe worth pulling that out in a way we can have all these functions just pick it up?  Nothing comes to mind, but maybe you have ideas...

Reply:
What is 'it' and 'that' in this context?

Reply:
Sorry - yes that was vague.  I more just meant that it looked like a common action was being taken at the beginning of "autotrack" methods and I was wondering if there was a way to structure it so that _every_ autotrack method got that data by default.

Reply:
I don't see an easy way to do this without doing some weird reflection-y thing or adding an additional function call to every instrumentation site.  I think just having each autotrack method delegating to this for base props at the beginning of this method is sufficient.